### PR TITLE
core: add operator commands in go

### PR DESCRIPTION
**Pull request by subhamkrai**
Signed-off-by: subhamkrai <srai@redhat.com>

### CODE REVIEW

File: pkg/k8sutil/operator.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func RestartDeployment(namespace, deploymentName string) {
+	// Get Kubernetes Client
+	_, clientset, _ := GetKubeClient()
+
+	deploymentsClient := clientset.AppsV1().Deployments(namespace)
+	data := fmt.Sprintf(`{"spec": {"template": {"metadata": {"annotations": {"kubectl.kubernetes.io/restartedAt": "%s"}}}}}`, time.Now().String())
+	_, err := deploymentsClient.Patch(context.TODO(), deploymentName, types.StrategicMergePatchType, []byte(data), v1.PatchOptions{})
```
Comment (travisn) [inline pkg/k8sutil/operator.go:35]: To restart the operator, I thought we could delete the pod, and using labels to match the pod. Patching this deployment actually restarts the operator?

Comment (subhamkrai) [reply pkg/k8sutil/operator.go:35]: yes, in my local testing it was restarting, I share the output today. 
We have 3ways to restart

1. using scale --replica=0
2. patch
3. delete

**Review by travisn — APPROVED** [review]

File: pkg/k8sutil/operator.go (hunk 1 above)
Comment (subhamkrai) [reply pkg/k8sutil/operator.go:35]: Screenshot from 2023-01-25 11-37-54

Comment (subhamkrai) [reply pkg/k8sutil/operator.go:35]: we can see that operator has restarted using the above command. Merging now.

**Pull request merged**